### PR TITLE
Regex not parsing response in `asn.py`

### PR DESCRIPTION
**Pull request by hmajid2301**
The regex expression was not parsing the response correctly in
asn:get_nets_radb. May issue seems to be the ipwhois data may have
changed no long ends in 
instead with \n.

Solves: #216

### CODE REVIEW

File: ipwhois/tests/test_asn.py [hunk 1]
```diff
@@ -136,8 +136,30 @@ def test_parse_fields_http(self):
...
+        for key, val in data.items():
+
+            log.debug('Testing: {0}'.format(key))
+            net = Net(key)
+            obj = IPASN(net)
+
+            try:
+
+                self.assertIsInstance(obj.lookup(), dict)
+                self.assertIsInstance(obj.lookup(asn_alts=['http']), dict)
```
Comment (secynic) [inline ipwhois/tests/test_asn.py]: asn_alts is removed next release, and has already been removed from the dev branch.

**Comment by secynic** [comment]
@hmajid2301 I meant to keep the tests, but just remove the line using asn_alts. Thanks!

File: ipwhois/tests/test_asn.py (hunk 1 above)
Comment (secynic) [reply ipwhois/tests/test_asn.py]: Please only remove line 154 (with the asn_alts argument).

**Comment by hmajid2301** [comment]
Done

**Comment by hmajid2301** [comment]
Any update on this?

**Comment by hmajid2301** [comment]
Any update on this?

File: ipwhois/tests/test_asn.py [hunk 2]
```diff
@@ -136,8 +136,29 @@ def test_parse_fields_http(self):
...
+        data_dir = path.dirname(__file__)
+
+        with io.open(str(data_dir) + '/asn.json', 'r') as \
+                data_file:
+            data = json.load(data_file)
+
+        for key, val in data.items():
+
+            log.debug('Testing: {0}'.format(key))
+        net = Net(key)
```
Comment (secynic) [inline ipwhois/tests/test_asn.py]: Line 148 and 149 should be indented to inside the for loop

**Pull request merged**